doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -235,25 +264,27 @@ For the User OS, we are using the same `Clear Linux`_ release version as the Ser
 
   .. code-block:: none
 
-     cd ~
-     wget https://download.clearlinux.org/releases/24040/clear/clear-24040-kvm.img.xz
-     unxz clear-24040-kvm.img.xz
+     cd /root/
+     wget https://download.clearlinux.org/releases/26440/clear/clear-26440-kvm.img.xz
+     unxz clear-26440-kvm.img.xz
 
-* Download the Production Kernel (PK) kernel
+* Download the "kernel-iot-lts2018" kernel
 
   .. code-block:: none
-
-     wget https://download.clearlinux.org/releases/24040/clear/x86_64/os/Packages/linux-pk414-standard-4.14.57-69.x86_64.rpm
-     rpm2cpio linux-pk414-standard-4.14.57-69.x86_64.rpm | cpio -idmv
+  
+     mkdir ~/uos-kernel-build
```

Comment:
Add a note to indicate this is done as a normal user (after the previous step which required him/her to become `root`)